app/policies/vet360_policy.rb

```diff
@@ -2,7 +2,12 @@
 
 Vet360Policy = Struct.new(:user, :vet360) do
   def access?
-    user.vet360_id.present?
+    if Flipper.enabled?(:va_v3_contact_information_service, user)
+      # user vet360_id is no longer needed for Contact Information Api V2
+      user.vet360_id.present? || user.idme_uuid.present? || user.logingov_uuid.present?
```

Comment:
Vet360_id is no longer needed. Idme_uuid and login_uuid can be used instead of vet360_id

Reply:
Do you mean vet360_id is not the only option? It seems like the first option in the statement and idme_uuid and logingov_uuid are fallback options.

Reply:
idme_uuid and login_uuid are not fall back options. All of these uuids can be used for the CI V2, [here](https://github.com/department-of-veterans-affairs/vets-api/blob/e8b0cb46e1db1407caf305b63e7340ee3511b075/lib/va_profile/v2/contact_information/service.rb#L215):
```
        def uuid_with_aaid
          return "#{@user.idme_uuid}^PN^200VIDM^USDVA" if @user.idme_uuid
          return "#{@user.logingov_uuid}^PN^200VLGN^USDVA" if @user.logingov_uuid

          "#{vet360_id}^PI^200VETS^USDVA"
        end
```

Reply:
CI V1, only the vet360_id could be used to access the api

Reply:
The vet360_id is not used for anything in this policy. It doesn't save the user, just checks to make sure the user has the uuids needed to make the calls.